x-pack/plugins/infra/common/http_api/metrics_explorer_views/v1/update_metrics_explorer_view.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
nit: This is similar to `create_metrics_explorer_view.ts`. I'm wondering if we could have types like `create_or_update_explorer_view.ts` types instead.

Reply:
I agree they are the same, but since the request payloads don't strictly represent the same structure in future, I preferred to keep them split for isolated responsibilities.